pkg/tsdb/elasticsearch/client/client.go

```diff
@@ -168,7 +168,7 @@ func (c *baseClientImpl) executeRequest(method, uriPath, uriQuery string, body [
 	return resp, nil
 }
 
-func (c *baseClientImpl) ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearchResponse, error) {
+func (c *baseClientImpl) ExecuteMultisearch(r *MultiSearchRequest) (*MultiSearchResponse, PluginError) {
```

Comment:
you shouldn't have to change this.  implementation of error will be PluginError from middleware.  if you want to change the source.  you can cast it to PluginError `pluginErr := error.(PluginError)`